front/src/app/LogsTable.tsx

```diff
@@ -28,12 +32,18 @@ export default function LogsTable({ logs }: Props) {
                 </thead>
                 <tbody>
                     {logs.map((log) => (
-                        <tr key={log.id} className="border-b border-gray-200">
+                        <tr
+                            key={log.id}
+                            className="hover:bg-gray-200 cursor-pointer"
+                            onClick={() => router.push(`logs/${log.id}`)}
```

Comment:
Is this typed? I wonder if theres a way we can make it typed 🤔 (the routes)

Reply:
done! 
![image](https://github.com/Theodo-UK/OmniLog/assets/57725347/a9eaec49-b843-425b-a19a-2215b486fc70)

Reply:
<3 you love to see it